base/gmp.jl

```diff
@@ -69,6 +69,93 @@ function __init__()
     end
 end
 
+
+module mpz
```

Comment:
Agree, most importantly because the convention can easily lead one to believe when perusing the code that `mpz` is an object rather than a module, which is quite confusing. So I think that writing `MPZ.add` would be less likely to be confusing.